WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -126,7 +127,6 @@
     @Inject SiteStore mSiteStore;
     @Inject MediaStore mMediaStore;
 
-
```

Comment:
This is so minor but thanks so much for doing this.

Reply:
lol is always bugs me when I see the double newlines. Happy to oblige with this 😄